### ISSUE TITLE
Unable to set Style in DataLabels

### ISSUE THREAD

**Issue by istibekesi** [vaadin/vaadin-charts-flow#403]
Thus, it is unable to set or disable textOutline via the API.

According to the JavaDoc in [DataLabel](https://github.com/vaadin/vaadin-charts-flow/blob/b32d9962485622ce3233a3f82b2f41d9d16b8463/addon/src/main/java/com/vaadin/flow/component/charts/model/DataLabels.java), setStyle(Style style) method `defaults to: {"color": "contrast", "fontSize": "11px", "fontWeight": "bold", "textOutline": "1px contrast" }`

However, the Style parameter is different from the one outlined in `plotOptions.[type].dataLabels.style`.

**Comment by alvarezguille** [comment]
You are right, current configuration object for style doesn't have all needed properties for datalabels. This needs new API. 

As a workaround you can extend Style and add the properties you need with their get/set and use it in `setStyle`, this way the missing properties will be serialized in the configuration.

What version of charts are you using? This workaround only applies to latest.

**Comment by istibekesi** [comment]
Thanks, suggested workaround works like a charm!
